src/libopencl.c

```diff
@@ -75,7 +77,7 @@ static int open_libopencl_so()
   {
     for(i=0; i<(sizeof(default_so_paths) / sizeof(char*)); i++)
     {
-      if(access_file(default_so_paths[i]))
+      if(default_so_paths[i][0] != '/' || access_file(default_so_paths[i]))
```

Comment:
There is a chance where relative path file might also be not present/accessible. We always need to check if the file is accessible

Reply:
Hmmm...... my initial changes were based on yang-le's fork (see #1), which doesn't have this problem. I will find a solution later.
P.S. I didn't meant to offend you, but why isn't #1 merged?

Reply:
It occured to me that `dlopen()` can access .so files in `LD_LIBRARY_PATH`, which `access_file()` can't. I think instead of checking its existence, maybe checking `so_handle != NULL` is better. I don't know if it is a good idea.